lib/accessory.js

```diff
@@ -54,15 +64,19 @@ class Accessory {
   }
 
   setOutlet (uuid, powered, done) {
-    this.remoteOutlet(uuid).update({desired_state: {powered}}, () => {
+	  var that = this;
+    this.remoteOutlet(uuid).update({desired_state: {powered}}, () => {	
+		that.outlets.map(function(outlet){
+			if(outlet.uuid == uuid) that.log(outlet.name + " set to state: " + powered);
+		})
       done(undefined, this.outlet(device.outlets, uuid));
     });
   }
 
   accessoryServices () {
     const responseHandler = done => {
       return (error, outlet) => {
-        return done(undefined, outlet.powered);
+		return done(undefined, (outlet)?outlet.powered:null);
```

Comment:
this could be shortened to:
```javascript
return done(undefined, outlet && outlet.powered);
```
(which should return `undefined` if `outlet` is not defined, I think